### PR TITLE
Add standard right click menu to desktop app

### LINKED ISSUE #3184: [Desktop] Right click doesn't work
We will need to figure out what elements should have right-click menus, and what options should be in those menus. E.g. I'm not sure how things like 'open in a new tab' should behave in an electron app.

### PR DESCRIPTION
**Pull request by mxstbr**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- desktop

Closes #3184

### CODE REVIEW

**Review by brianlovin — DISMISSED** [review]

File: desktop/src/main.js [hunk 1]
```diff
@@ -12,7 +13,6 @@ const CONFIG = require('./config');
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 // eslint-disable-next-line
-let win;
```
Comment (brianlovin) [inline desktop/src/main.js]: Why did you remove this line?

Comment (mxstbr) [reply desktop/src/main.js]: Because `win` is unused in the entirety of the file lol

Comment (mxstbr) [reply desktop/src/main.js]: I mean I can leave it there but the variable is unused.

Comment (brianlovin) [reply desktop/src/main.js]: I know it's unused - the comment above says to do that though? Idk
